v1/personio.go

```diff
@@ -443,10 +443,10 @@ func (personio *Client) GetTimeOffs(start *time.Time, end *time.Time, offset int
 
 		query := req.URL.Query()
 		if start != nil {
-			query.Add("start_date", start.Format(time.RFC3339))
+			query.Add("start_date", start.Format("2006-01-02"))
```

Comment:
Would you like to:
* make this a package private variable at the top of this file (eg. `const dateFormat = "2006-01-02"`)
* then also use that variable in `personio_test.go` (lines 103, 109)
?

Reply:
Maybe also add something like the following to the CHANGELOG.md under section `## [Unreleased]`:

```
### Fixed

- Fixed `GetTimeOffs()` date argument format (YYYY-MM-DD)
```